docs/dictionary/command/ask.lcdoc

```diff
@@ -56,4 +74,8 @@ The ability to provide formatted text for the prompt was introduced in version 2
 Changes:
 The ability to specify an <iconType> was added in version 2.0. In previous versions, no icon was displayed.
 
-References: dialogData (property), gRevAppIcon (keyword), it (keyword), gRevProfileReadOnly (keyword), gRevSmallAppIcon (keyword), gRevAppIcon keyword (keyword), gRevSmallAppIcon keyword (keyword), answer (command), ask file (command), ask password (command), modal (command), sheet (command), ask file with type (command), result (function), return (glossary), OS X (glossary), variable (glossary), handler (glossary), modal dialog box (glossary), dialog box (glossary), command (glossary), function (control_st)
+References: dialogData (property), gRevAppIcon (keyword), it (keyword), gRevProfileReadOnly (keyword), gRevSmallAppIcon (keyword), 
+gRevAppIcon keyword (keyword), gRevSmallAppIcon keyword (keyword), answer (command), ask file (command), ask password (command), 
```

Comment:
@asayd Oops, should have noticed before, these two 
```
gRevAppIcon keyword (keyword), gRevSmallAppIcon keyword (keyword)
```
can also be removed from the references.